covalent/executor/executor_plugins/dask.py

```diff
@@ -66,13 +68,20 @@ def __init__(
         log_stderr: str = "stderr.log",
         conda_env: str = "",
         cache_dir: str = "",
+        workdir: str = "",
         current_env_on_conda_fail: bool = False,
     ) -> None:
         if not cache_dir:
-            cache_dir = os.path.join(
-                os.environ.get("XDG_CACHE_HOME") or os.path.join(os.environ["HOME"], ".cache"),
-                "covalent",
-            )
+            cache_dir = _EXECUTOR_PLUGIN_DEFAULTS["cache_dir"]
+
+        if not workdir:
+            try:
+                workdir = get_config("executors.dask.workdir")
+            except KeyError:
+                workdir = _EXECUTOR_PLUGIN_DEFAULTS["workdir"]
+                info_msg = f"Couldn't find `executors.dask.workdir` in config, creating a default one at {workdir}"
+                app_log.info(info_msg)
+                print(info_msg)
```

Comment:
Let's not print anything 😅 and make the log debug instead.

Reply:
done